doc/_dev-guide/chapter_4/section_2/README.md

```diff
@@ -11,16 +11,20 @@ id: dev-platform-build-macos
 
 At present we are not supporting macOS {{ site.product.short_name }} on our [[official repository|gh-syslog-ng]] on GitHub. However, you can compile {{ site.product.short_name }} yourself following this guide.
 
-**Note:** The guide is tested on ARM macOS Sonoma 14.2.1, Ventura 13.4, and Intel macOS Monterey 12.6.6 machines, we do our bests to keep it update, but your actual system may require additional steps or slightly different settings.
+**Note:** The guide is tested on ARM macOS Sonoma 14.6.1, Ventura 13.4, and Intel macOS Monterey 12.6.6 machines, we do our bests to keep it update, but your actual system may require additional steps or slightly different settings.
 {: .notice}
 
 ## Compiling from source
 
-Like every project {{ site.product.short_name }} also uses different libraries and build-systems that must be installed for compiling and running properly. These dependencies can be satisfied by compiling every-each libs and tools manually, but it might be preferred to do it the easy way. Homebrew is a package manager for macOS that has great community and support. You can also use it to install the dependencies you need.
+Like every project {{ site.product.short_name }} also uses different libraries and build-systems (a.k.a dependencies) that must be installed for compiling and running properly.
 
 ### Dependencies
 
-1. [[Install Homebrew|dev-inst-macos#homebrew]] on your system.
+The above mentioned dependencies can be satisfied by compiling every-each libs and tools manually, but it might be preferred to do it the easy way. Homebrew and MacPorts are package managers for macOS which has great community and support. You can also use them to install the dependencies you need.
```

Comment:
```suggestion
The above mentioned dependencies can be satisfied by compiling every-each libs and tools manually, but it might be preferred to do it the easy way. Homebrew and MacPorts are package managers for macOS which have great communities and support. You can also use them to install the dependencies you need.
```